index.html

```diff
@@ -238,6 +238,23 @@ <h3 class="lang-header">Languages</h3>
         </ul>
       </div>
     </section>
+    <section class="aboutme contact-me">
+      <div class="bio">
+        <img class="dot1" src="./icons/dot1.png" alt="" />
```

Comment:
- [ ] If you notice the images in the contact form are purposely for decoration. Therefore they should be added using the CSS styling instead of `img`. Kindly ensure to fix this, so that the contact form is more professional and matches the Figma design ☺️ .